packages/core/src/shared/utilities/collectionUtils.ts

```diff
@@ -329,6 +330,33 @@ export function partialClone(obj: any, depth: number = 3, omitKeys: string[] = [
     return clonedObj
 }
 
+type inspectOptions = Partial<{
+    depth: number
+    omitKeys: string[]
+    replacement: any
+    showHidden: boolean
+    color: boolean
```

Comment:
Is it clearer if we add `?` to the fields instead of using Partial? I'm not certain about what's most common in the typescript ecosystem, but I would guess that Partial is used to modify an existing type, not one that we are defining inline.

Reply:
```ts
import { InspectOptions } from 'util';

type inspectOptions = Partial<InspectOptions>;
```